libview/ev-view.c

```diff
@@ -3516,6 +3521,10 @@ ev_view_scroll_event (GtkWidget *widget, GdkEventScroll *event)
 		        case GDK_SCROLL_LEFT:
 				ev_view_previous_page (view);
 				break;
+#if GTK_CHECK_VERSION (3, 0, 0)
+                        case GDK_SCROLL_SMOOTH:
+                                g_assert_not_reached ();
```

Comment:
This is going to make Atril crash on receiving ```GDK_SCROLL_SMOOTH``` event (which can happen in GTK+3 build, I already encountered such crash in some other app). Need to backport proper handling from upstream...